pkg/package-installer/windows-subprocess.go

```diff
@@ -165,12 +163,8 @@ type serviceInstaller struct {
 	responseChannel <-chan responseMessage
 }
 
-func NewServiceInstaller() (PackageInstaller, error) {
-	pipeName := *flagServicePipeName
-
-	if pipeName == "" {
-		pipeName = fmt.Sprintf("\\\\.\\pipe\\UnityInstaller-%s", uuid.New().String())
-	}
+func NewServiceInstaller(logger logr.Logger) (PackageInstaller, error) {
+	pipeName := fmt.Sprintf("\\\\.\\pipe\\UnityInstaller-%s", uuid.New().String())
```

Comment:
```suggestion
	pipeName := fmt.Sprintf(`\\.\pipe\UnityInstaller-%s`, uuid.New().String())
```

I learned about these today, from you/Jordan! So I'm gonna suggest them! :D